bundles/test/org.eclipse.smarthome.test/src/main/groovy/org/eclipse/smarthome/test/storage/VolatileStorage.java

```diff
@@ -12,6 +12,8 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 import org.eclipse.jdt.annotation.NonNull;
```

Comment:
There is still @NonNull in use in this class, which should be superseded by @NonNullByDefault.